mapeditor/mapcontroller.cpp

```diff
@@ -143,7 +143,9 @@ void MapController::commitObjectErase(int level)
 	}
 	else if (selectedObjects.size() == 1)
 	{
-		_map->getEditManager()->removeObject(*selectedObjects.begin());
+		_map->getEditManager()->removeObject(selectedObjects.front());
+		//invalidate tiles under object
+		_mapHandler->invalidate(_mapHandler->geTilesUnderObject(selectedObjects.front()));
 	}
```

Comment:
What the heck happened here. This didn't even compile as set has no front() method. Also, this version doesn't work correctly either.

Reply:
Oops, it's my fault- was trying to resolve merge conflicts